### PR TITLE
Update test to use testify framework

### PR DESCRIPTION
**Pull request by italolelis**
#### Summary
Replaced the standard go testing package by the testify package for the file `model/command_webhook_test.go` as stated in #12553.
I've tried to replicated the exact same functionality as it was before. Please let me know if it's not up to the standards and I can fix it.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/12553

### CODE REVIEW

File: model/command_webhook_test.go [hunk 1]
```diff
@@ -44,11 +44,13 @@ func TestCommandWebhookIsValid(t *testing.T) {
...
 		err := h.IsValid()
-		if test.ExpectedError == "" && err != nil {
-			t.Fatal("hook should be valid")
-		} else if test.ExpectedError != "" && test.ExpectedError != err.Id {
-			t.Fatal("expected " + test.ExpectedError + " error")
+
+		if test.ExpectedError == "" {
+			assert.Error(t, err, "hook should be valid")
+		} else {
+			assert.Equal(t, test.ExpectedError, err.Id, "expected "+test.ExpectedError+" error")
```
Comment (lieut-data) [inline model/command_webhook_test.go]: This was an existing issue, but if `err` is `nil`, we'll get a panic here. Can we `require.NotNil` here?

Comment (italolelis) [reply model/command_webhook_test.go]: Ah true, sorry I missed that. I'll add the `require.NotNil`

**Comment by italolelis** [comment]
@lieut-data I've applied the changes, hopefully it's ok now 😄

**Comment by jespino** [comment]
No QA needed for this PR, only modifies unit tests.

**Pull request merged**